src/components/profile/UserProfilePage/style.module.scss

```diff
@@ -0,0 +1,215 @@
+        align-items: center;
+        flex-direction: column;
+      }
+
+      .profilePic {
+        border-radius: 4rem;
+        flex-shrink: 0;
+        height: 7rem;
+        overflow: hidden;
+        position: relative;
+        width: 7rem;
+      }
+
+      .cardName {
+        align-items: flex-start;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+
+        @media (max-width: vars.$breakpoint-md) {
+          align-items: center;
+          flex-direction: column;
+          
+          h1 {
+            text-align: center;
+          }
+        }
+      }
+
+      .cardRank {
+        align-items: flex-end;
+        display: flex;
+        flex-direction: column;
+        flex-grow: 1;
+        gap: 0.5rem;
+        justify-content: flex-end;
+
+        @media (max-width: vars.$breakpoint-md) {
+          align-items: center;
+          color: var(--theme-text-on-background-2);
+          flex-direction: column;
+        }
+
+        .rank {
+          color: var(--theme-text-on-background-1);
+          display: flex;
+          font-size: 1.25rem;
+          font-weight: 500;
+
+          @media (max-width: vars.$breakpoint-md) {
+            text-align: center
+          }
+        }
+
+        .points {
+          color: var(--theme-text-on-background-1);
+          font-size: 1.125rem;
+          font-weight: 400;
+          line-height: 1.25;
+          text-align: end;
+          
+          @media (max-width: vars.$breakpoint-md) {
+            text-align: center;
+          }
+
+          svg {
+            height: 1.125rem;
+            width: 1.125rem;
+          }
+        }
+      }
+    }
+  }
+
+  .section {
+    border: 2px solid #808184;
+    border-radius: 0.75rem;
+    padding: 1rem 1.5rem;
+  }
+
+  .progressSection {
+    display: flex;
+    flex-direction: column;
+    gap: 0.9375rem;
+
+    .progressInfo {
+      display: flex;
+      flex-flow: row wrap;
+      justify-content: space-between;
+    }
+
+    .progressBar {
+      background-color: var(--theme-text-on-background-3);
+      border-radius: 0.5rem;
+      height: 1.25rem;
+      width: 100%;
+
+      .inner {
+        background-color: var(--theme-primary-2);
+        border-radius: 0.5rem;
+        height: 100%;
+        transition: width 1s ease;
+      }
+    }
+
+    p {
+      @media (max-width: vars.$breakpoint-md) {
+        text-align: center;
+      }
+    }
+  }
+
+  .aboutSection {
+    display: grid;
+    grid-template-columns: 20rem 1fr;
+    position: relative;
+
+    @media (max-width: vars.$breakpoint-md) {
+      display: flex;
+      flex-direction: column;
+      gap: 1.875rem;
+    }
+
+    .aboutMeSection {
+      display: grid;
+      gap: 0.56rem;
+      grid-template-columns: auto 1fr;
+      margin: 0.81rem 0;
+
+      .icon {
+        height: 1.313rem;
+        width: 1.313rem;
+      }
+
+    }
+
+    svg {
+      height: 2rem;
+      width: 2rem;
+    }
+
+    .bioSection {
```

Comment:
(This is also an issue with the `Preview` component) It appears newlines in the `bio` get removed. You could add

```suggestion
    .bioSection {
      white-space: pre-wrap;
```

so the text shows the same as how it appears in the text input (keeps line breaks, duplicate spaces, but also allows text wrapping)

Reply:
you should also add `word-break: break-word;` or whatever it's called, so

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/84ca3f59-9487-43a8-8fbc-036d49603f67) ![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/305173f9-ae08-4849-aea3-f994a4e03196)

Reply:
Good catch! Implemented in latest commit https://github.com/acmucsd/membership-portal-ui-v2/commit/8a0198fd280f38cb33d2fb9007c79be99804e06c 👍